plutus-core/plutus-core/src/PlutusCore/Evaluation/Result.hs

```diff
@@ -75,22 +77,28 @@ instance MonadError () EvaluationResult where
 
 instance Applicative EvaluationResult where
     pure = EvaluationSuccess
+    {-# INLINE pure #-}
```

Comment:
Those definitions do inline on their own, but it never hurts to be precise (and it does occasionally hurt not to be precise).